resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/ApplicationMaster.scala

```diff
@@ -788,9 +788,9 @@ private[spark] class ApplicationMaster(
 
     override def onStart(): Unit = {
       driver.send(RegisterClusterManager(self))
-      // if deployment mode for yarn Application is client
+      // if deployment mode for yarn Application is managed client
       // then send the AM Log Info to spark driver
-      if (!isClusterMode) {
+      if (!isClusterMode && !sparkConf.get(YARN_UNMANAGED_AM)) {
```

Comment:
This causes the entire process to not show up in the executor page then?

Reply:
Only the driver process not show:
<img width="1621" alt="image" src="https://github.com/apache/spark/assets/5399861/337a363d-e1b9-4ca3-b25a-b1f57c2cfec9">

Reply:
sorry, I don't know what you mean by that? is this the after picture and if so what was the before picture?

Reply:
There is no difference on the executor page, but there will be an error when starting the driver.